lisa/tools/iperf3.py

```diff
@@ -340,6 +343,7 @@ def create_iperf_udp_performance_message(
         connections_num: int,
         environment: "Environment",
         test_case_name: str,
+        result: "TestResult",
```

Comment:
The environment should be attached to the result already. We can try to refine it better to remove environment from the message. So update the `create_perf_message` to accept test result, and retrieve environment from the test result

Reply:
update